app/ante.go

```diff
@@ -66,29 +66,31 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 	// 	sigVerifyDecorator = ante.NewBatchSigVerificationDecorator(options.BatchVerifier, sequentialVerifyDecorator)
 	// }
 
-	anteDecorators := []sdk.AnteDecorator{
-		ante.NewSetUpContextDecorator(), // outermost AnteDecorator. SetUpContext must be called first
+	anteDecorators := []sdk.AnteFullDecorator{
+		sdk.DefaultWrappedAnteDecorator(ante.NewSetUpContextDecorator()), // outermost AnteDecorator. SetUpContext must be called first
```

Comment:
nit - could probably create a function for this i.e. `wrapDecorator(decs []Decorator) ([]WrappedDecorator, err)`

Reply:
I think its better to explicitly wrap each ante handler, because otherwise future changes may change order of the antehandlers without properly changing the order of dependency generators, and that could potentially result in unintended behavior